geth-utils/gethutil/mpt/witness/gen_witness_transactions_test.go

```diff
@@ -16,11 +16,54 @@ import (
 )
 
 func TestTransactions(t *testing.T) {
-	t.Skip("failing test")
 	txs := make([]*types.Transaction, 70)
 	key, _ := crypto.GenerateKey()
 	signer := types.LatestSigner(params.TestChainConfig)
 
+	/*
```

Comment:
This comment looks long.
Can we move this paragraph to line 17 so that it becomes the function documentation?